tests/eggcc-extraction.egg

```diff
@@ -1704,6 +1704,6 @@
 (let v103 (Func v0 v4 v5 v102))
 
 (run-schedule
-        (repeat 5 (saturate fast-analyses)
+        (repeat 2 (saturate fast-analyses)
```

Comment:
Can you put a comment here that explains how this number affects CI performance? e.g. from your commit history you ran it at 2, 3, and 5, can you write down what the proportional slowdown is?

Reply:
Sounds good, added